### PR TITLE
Translating and fixing some pt-BR translations

### PR DESCRIPTION
**Pull request by marciioluucas**

### CODE REVIEW

**Comment by peluprvi** [comment]
@marciioluucas , thank you for helping us translate Vuetify!
I'm moving to a new home (still without internet in it). Sorry for not merge it and not finish the review yet. I'll continue to review as soon as I can.

PS: Please, remember to keep comments, commits, and all github messages in English so that everyone can undestand and contribute to discussions.

File: lang/pt-br/getting-started/WhyVuetify.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+export default {
+  header: 'Qual é a diferença?',
+  headerText: '',
+  toc: [
+    {
+      text: `Qual é a diferença?`,
+      href: 'porque-vuetify'
```
Comment (jacekkarczmarczyk) [inline lang/pt-br/getting-started/WhyVuetify.js]: hrefs should not be translated

Comment (ackzell) [reply lang/pt-br/getting-started/WhyVuetify.js]: I may or may not have translated those in my fork 😶. Was this documented somewhere? Would be good to note it in the readme or something 😅

Comment (KaelWD) [reply lang/pt-br/getting-started/WhyVuetify.js]: No, we need to remove them at some point anyway for #242

**Pull request merged**